### PR TITLE
Update babel-generator's README

### PR DESCRIPTION
**Pull request by jddxf**
Fix the broken code sample in section `AST from Multiple Sources`

| Q                        | A 
| ------------------------ | ---
| Patch: Bug Fix?          | 
| Major: Breaking Change?  | 
| Minor: New Feature?      | 
| Deprecations?            | 
| Spec Compliancy?         | 
| Tests Added/Pass?        | 
| Fixed Tickets            | 
| License                  | MIT
| Doc PR                   | Yes
| Dependency Changes       |

### CODE REVIEW

**Comment by existentialism** [comment]
@jddxf nice catch, thanks!

Can you also make some tweaks to the instructional text above the example to match these changes?

**Comment by jddxf** [comment]
@existentialism Actually I think there's nothing wrong with the text.Though it could be improved. I'll think about it tomorrow.

**Comment by jddxf** [comment]
I think the first point is not relevant as you'll always need that to enable sourcemap whether the AST is from multiple sources or not, so I removed it. 

Also I didn't mention how to construct a new AST from multiple sources as the section name `AST from Multiple Sources` suggests we already have such an AST.

Hope it makes sense.

File: packages/babel-generator/README.md [hunk 1]
```diff
@@ -68,14 +63,14 @@ import generate from 'babel-generator';
 
 const a = 'var a = 1;';
 const b = 'var b = 2;';
-const astA = parse(a, { filename: 'a.js' });
-const astB = parse(b, { filename: 'b.js' });
+const astA = parse(a, { sourceFilename: 'a.js' });
```
Comment (xtuc) [inline packages/babel-generator/README.md]: nit: It's `sourceFileName`

Comment (jddxf) [inline packages/babel-generator/README.md]: See https://github.com/babel/babylon/blob/master/README.md#options.

Comment (xtuc) [inline packages/babel-generator/README.md]: Yes! you'r right. Sorry, I was looking at Babel-generator's options.

**Pull request merged**